processor/src/host/dsa.rs

```diff
@@ -1,10 +1,11 @@
 use alloc::vec::Vec;
 
+#[cfg(feature = "std")]
 use vm_core::{
     crypto::dsa::rpo_falcon512::{Polynomial, SecretKey},
     utils::Deserializable,
-    Felt, Word,
 };
```

Comment:
nice: since there is only one function that uses these, I'd probably move the imports into the function itself (e.g., to line 28).

Reply:
done